qokit/portfolio_optimization.py

```diff
@@ -88,61 +90,58 @@ def get_data(N, seed=1, real=False) -> tuple[float, float]:
     https://github.com/Qiskit/qiskit-finance/blob/main/docs/tutorials/11_time_series.ipynb
     """
     import datetime
-    from qiskit_finance.data_providers import RandomDataProvider, YahooDataProvider
 
-    tickers = []
-    for i in range(N):
-        tickers.append("t" + str(i))
-    if real is False:
-        data = RandomDataProvider(tickers=tickers, start=datetime.datetime(2016, 1, 1), end=datetime.datetime(2016, 1, 30), seed=seed)
-    else:
-        stock_symbols = [
-            "AAPL",
-            "GOOGL",
-            "AMZN",
-            "MSFT",
-            "TSLA",
-            "NFLX",
-            "NVDA",
-            "JPM",
-            "V",
-            "JNJ",
-            "WMT",
-            "PG",
-            "MA",
-            "UNH",
-            "HD",
-            "DIS",
-            "BRK-B",
-            "VZ",
-            "KO",
-            "MRK",
-            "INTC",
-            "CMCSA",
-            "PEP",
-            "PFE",
-            "CSCO",
-            "XOM",
-            "BA",
-            "MCD",
-            "ABBV",
-            "IBM",
-            "GE",
-            "MMM",
-        ]
-
-        data = YahooDataProvider(
-            tickers=stock_symbols[:N],
-            start=datetime.datetime(2020, 1, 1),
-            end=datetime.datetime(2020, 1, 30),
-            # end=datetime.datetime(2021, 1, 1),
-        )
+    from qokit.yahoo import YahooDataProvider
+
+    stock_symbols = [
+        "AAPL",
+        "GOOGL",
+        "AMZN",
+        "MSFT",
+        "TSLA",
+        "NFLX",
+        "NVDA",
+        "JPM",
+        "V",
+        "JNJ",
+        "WMT",
+        "PG",
+        "MA",
+        "UNH",
+        "HD",
+        "DIS",
+        "BRK-B",
+        "VZ",
+        "KO",
+        "MRK",
+        "INTC",
+        "CMCSA",
+        "PEP",
+        "PFE",
+        "CSCO",
+        "XOM",
+        "BA",
+        "MCD",
+        "ABBV",
+        "IBM",
+        "GE",
+        "MMM",
+    ]
+
+    data = YahooDataProvider(
+        tickers=stock_symbols[:N],
+        start=datetime.datetime(2020, 1, 1),
+        end=datetime.datetime(2020, 1, 30),
+    )
 
     data.run()
     # use get_period_return_mean_vector & get_period_return_covariance_matrix to get return!
     # https://github.com/Qiskit/qiskit-finance/blob/main/docs/tutorials/01_portfolio_optimization.ipynb
-    means = data.get_period_return_mean_vector()
-    cov = data.get_period_return_covariance_matrix()
+    # means = data.get_period_return_mean_vector()
```

Comment:
Can you please remove commented out code?

Reply:
Done!